website/docs/terms/surrogate-key.md

```diff
@@ -30,17 +30,17 @@ Let’s take this to an example. Below, there is a table you pull from an ad pla
 
 <table>
   <tr>
-   <td>calendar_date
+   <td><b>calendar_date</b>
```

Comment:
Should these headings also be in backticks like they are in the paragraphs? I haven't seen too many tables in the docs, but it might help distinguish between code/not code.

Reply:
Hey they shouldn't be code, but we wanted to bold the column headers to make them stand out more!